context/index.d.ts

```diff
@@ -0,0 +1,17 @@
+import type { ReadableAtom } from '../atom'
+
+// This isn't a part of the public API for now
+type Context = unknown
```

Comment:
We need a public API for dev tools:

1. Get all active stores on a current page
2. Including all stores from store creators like `User(id)` from Logux Client or [MapCreator](https://github.com/nanostores/nanostores/blob/next/map-creator/index.d.ts)

Reply:
Yeah, sure. That's a TBD.
The way I currently store the assoc between store and its value is `Map<Store, StoreValue>`. Main problem is that it's not _serializable_. I don't have a solution for this yet, but that will definitely change the way we store contexts, hence change the public API for contexts and context getters.